src/app.js

```diff
@@ -320,6 +347,8 @@ var run = function () {
 
     currentFile = file
 
+    files.event.trigger('fileFocus', [file])
```

Comment:
Please add a simple function to the files module that will trigger the event. Triggering events of other modules should not be possible.